packages/expo-updates/android/src/main/java/expo/modules/updates/db/UpdatesDatabase.kt

```diff
@@ -161,5 +162,23 @@ abstract class UpdatesDatabase : RoomDatabase() {
         }
       }
     }
+
+    val MIGRATION_8_9: Migration = object : Migration(8, 9) {
+      override fun migrate(database: SupportSQLiteDatabase) {
+        // https://www.sqlite.org/lang_altertable.html#otheralter
+        try {
+          database.execSQL("PRAGMA foreign_keys=OFF")
+          database.beginTransaction()
```

Comment:
Good point. All the other ones have it here but I'll change this one to start doing it this way going forward.

Reply:
I wonde if adding a small static inner class in this module that implements AutoCloseable wouldn't be overkill. It would take a database object and take care of turning off and on the FKs and also take care of beginning and ending the transaction.

```
MigrationTransaction(database).use {
  database...
  database.setTransactionSuccessful()
}
```

Reply:
Will investigate in a separate PR. https://linear.app/expo/issue/ENG-2628/investigate-migrationtransaction-auto-closable